File: TLX-Toki/Pascal/MembalikYgTblk.pas
function reverse(x:longint): longint;
var
temp, ret : longint;
begin
  temp := x;
  ret := 0;
  while (temp > 0) do begin
    ret := (ret * 10) + (temp mod 10);
	temp := temp div 10;
  end;
  reverse := ret;
end;

var
  a,b,c : longint;
begin
  readln(a,b);
  a := reverse(a);
  b := reverse(b);
  c := a + b;
  c := reverse(c);
  writeln(c);
end.

File: TLX-Toki/CPP/MembalikYgTblk.cpp
#include<stdio.h>
using namespace std;

long reverse(long x) 
{
    long temp, ret;
	temp = x;
    ret  = 0;

    while (temp > 0) {
        ret  = (ret * 10) + (temp % 10);
        temp = temp / 10;
    }
    return(ret);
}

int main()
{
	long A, B, C;
	scanf("%ld %ld",&A,&B);
	A = reverse(A);
	B = reverse(B);
	C = A + B;
	C = reverse(C);
	printf("%ld\n",C);
}